releasetasks/templates/l10n.yml.tmpl

```diff
@@ -135,6 +135,15 @@
             {% if running_tests is defined %}
             task_name: "{{ '{}_beetmover_candidates_{}'.format(buildername, chunk) }}"
             {% endif %}
+            build_props:
+                locales:
+{% for locale in our_locales %}
+                    - {{ locale }}
```

Comment:
That means if we have to repack a locale it will generate another Pulse message with this locale only? Or do we have to re-run the whole repack job?